app/views/directives/create-secret.html

```diff
@@ -234,6 +234,11 @@
             required>
         </div>
       </div>
+      <div class="has-error" ng-show="secretForm.dockerEmail.$invalid && !secretForm.dockerEmail.$pristine">
```

Comment:
I'd check specifically for `secretForm.dockerEmail.$error.email` and have separate messages for invalid email values and missing required values.